pythonforandroid/recipes/jpeg/__init__.py

```diff
@@ -35,10 +34,9 @@ def build_arch(self, arch):
                     '-DCMAKE_POSITION_INDEPENDENT_CODE=1',
                     '-DCMAKE_ANDROID_ARCH_ABI={arch}'.format(arch=arch.arch),
                     '-DCMAKE_ANDROID_NDK=' + self.ctx.ndk_dir,
-                    '-DCMAKE_C_COMPILER={toolchain}/bin/clang'.format(
-                        toolchain=env['TOOLCHAIN']),
-                    '-DCMAKE_CXX_COMPILER={toolchain}/bin/clang++'.format(
-                        toolchain=env['TOOLCHAIN']),
+                    '-DCMAKE_C_COMPILER={cc}'.format(cc=arch.get_clang_exe()),
+                    '-DCMAKE_CXX_COMPILER={cc_plus}'.format(
+                        cc_plus=arch.get_clang_exe(plus_plus=True)),
```

Comment:
So you actually did find a bug already while covering it with your tests? :clap:

Reply:
Aaa...this is more a little refactor (since at the time we changed JPEG to be built with Cmake, the function `arch.get_clang_exe` does not existed, we were forced to got these values in `get_recipe_env`, since our default compiler was `gcc`, but with the NDK-r19 migration thing, we use `clang`, so we can make use of  `arch.get_clang_exe` and remove some lines :smile:)